content/docs/1.4.0/advanced-resources/_index.md

```diff
@@ -1,4 +1,22 @@
 ---
 title: Advanced Resources
 weight: 70
----
\ No newline at end of file
+---
+
+* [Backing Image](./backing-image)
+* [Fast Replica Rebuild](./fast-replica-rebuild)
+* [Orphaned Data Cleanup](./orphaned-data-cleanup)
+* [ReadWriteMany (RWX) Volume](./rwx-workloads)
+* [Restore cluster with a Rancher snapshot](./rancher-cluster-restore)
+* [Snapshot Data Integrity Check](./snapshot-data-integrity-check)
+* [Use Longhorn Volume as an iSCSI Target](./iscsi)
+* [Migrating from the Flexvolume Driver to CSI](./migrating-flexvolume)
+* [Configuring Defaults for Nodes and Disks](./default-disk-and-node-config)
+* [Troubleshooting](./troubleshooting)
+* [Support Bundle](./support-bundle)
+* [Deploy](./deploy/)
```

Comment:
nit: do we need the trailing slash? This seems a bit inconsistent.

Reply:
Just tell it contains sub pages, but it could be removed.
Will do.

Reply:
Removed. Thanks